docker-compose/_posts/2017-04-03-launching-your-first-environment.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using Docker Compose With Runnable
+shortTitle: Launching your first environment
+step: 3
+tags:
+- docker
+description: Using Docker Compose With Runnable
+---
+
+
+Runnable makes it easy to launch containers from your compose file. 
+
+### Configuring your first repository
+
+
+Let's walkthrough configuring an example repository. The repository we will configure has the following Docker compose file:
+  ![GitHub Organization People]({{ site.baseurl }}/images/compose_config_composefile.png)
```

Comment:
Those are used to describe what the image is. In case it doesn't load or whatever.